source/includes/_01-your-first-backend-in-10-minutes.md

```diff
@@ -0,0 +1,204 @@
+- [Conclusion](#10-conclusion)
+
+## 1. Create project
+First of all, we need to create a base Booster project
+
+`boost new:project <project-name>`
+
+You will need to answer a few questions in order to configure the project. One of these steps will require that a provider is selected, we recommend selecting AWS. 
+
+Once the project has been successfully created, you will need to move to the new directory. Ideally, you would also open the project in your most preferred IDE, e.g. Visual Studio Code.
+
+## 2. First command
+We will now define our first command, which will allow us to create Posts.
+
+`boost new:command CreatePost --fields postId:UUID title:string content:string author:string`
+
+These commands creates most of the code for us, which can be seen in `<project-root>/src/commands/CreatePost.ts`
+
+However, we still need to define a couple of things in this file:
+- Who can trigger our command
+- What events should be triggered
+
+For now, we will set the `authorize` configuration to `all`, so anyone can trigger it.
+
+Additionally, the current `CreatePost` command will not trigger any event, so we will have to come back later to set the event that this command will fire up.
+
+## 3. First event
+In this type of backend architectures, events can be triggered by commands or by other events. We will create an event that defines a `Post` creation. 
+
+`boost new:event PostCreated --fields postId:UUID title:string content:string author:string`
+
+You will realize that a new file has been created, `<project-root>/src/events/PostCreated.ts`
+
+There is one small thing that we have to define in the above file, which is the returned value for `EntityID()`. We will set the post `UUID`. It should look like this:
+```typescript
+public entityID(): UUID {
+    return this.postId
+  }
+```
+
+Now we can go back to the command we created before and add our new event `PostCreated` to the register of events. Your `handle` should look like this:
+```typescript
+public handle(register: Register): void {
+    register.events(new PostCreated(this.postId, this.title, this.content, this.author))
+  }
+```
+
+## 4. First entity
+We have now created a command and an event, however, we do not have any data representation of a `Post`. As a result, we will create an `entity`.
+
+`boost new:entity Post --fields title:string content:string author:string --reduces PostCreated`
+
+Another file has been created in your project, `<project-root>/src/entities/Post.ts`
+
+The `Post` entity has been created but no projection has been defined, so we will do so:
+
+```typescript
+@Reduces(PostCreated)
+public static projectPostCreated(event: PostCreated, currentPost?: Post): Post {
+return new Post(event.postId, event.title, event.content, event.author)
+}
+```
+
+In the future, we may want to project events for this `Post` entity that require retrieving current `Post` values. In that case we would need to make use of `currentPost`. 
```

Comment:
```suggestion
In the future, we may want to reduce events for this `Post` entity that require retrieving current `Post` values. In that case we would need to make use of `currentPost`.

Note that it is nullable, meaning that it can be `undefined` in some cases. (Like when there is no `Post` created, and it is the first time the reducer is being executed)
```